src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -327,20 +331,20 @@ def test_storage_azcopy_blob_account(self, resource_group, first_account, second
 
         # Copy a single blob to another single blob
         self.cmd('storage copy --source-account-name {} --source-container {} --source-blob {} \
-                 --destination-account-name {} --destination-container {}'
+                 --destination-account-name {} --destination-container {} --preserve-s2s-access-tier false'
                  .format(first_account, first_container, 'readme', second_account, second_container))
         self.cmd('storage blob list -c {} --account-name {}'
                  .format(second_container, second_account), checks=JMESPathCheck('length(@)', 1))
 
         # Copy an entire directory from blob virtual directory to another blob virtual directory
         self.cmd('storage copy --source-account-name {} --source-container {} --source-blob {} \
-                 --destination-account-name {} --destination-container {} --recursive'
+                 --destination-account-name {} --destination-container {} --recursive --preserve-s2s-access-tier false'
                  .format(first_account, first_container, 'apple', second_account, second_container))
         self.cmd('storage blob list -c {} --account-name {}'
                  .format(second_container, second_account), checks=JMESPathCheck('length(@)', 11))
 
         # Copy an entire storage account data to another blob account
-        self.cmd('storage copy --source-account-name {} --destination-account-name {} --recursive'
+        self.cmd('storage copy --source-account-name {} --destination-account-name {} --recursive --preserve-s2s-access-tier false'
```

Comment:
Do we support abbreviation? Parameter name is long. I'm not willing to use it when I see such a  long parameter.

Reply:
It is not used very much, just for a special case. And it may be removed in future.